src/main/java/org/prebid/server/auction/BidResponseCreator.java

```diff
@@ -170,6 +170,7 @@ private static int validateTruncateAttrChars(int truncateAttrChars) {
         final long auctionTimestamp = auctionTimestamp(auctionContext);
         final Account account = auctionContext.getAccount();
         final List<String> debugWarnings = auctionContext.getDebugWarnings();
+        final String auctionId = bidRequest.getId();
```

Comment:
i'd say it should be named as `requestId` or `bidRequestId` since `auctionId` is event-specific term and core should not care how it used in events.